packages/launchpad/src/migration/RenameSpecsAuto.vue

```diff
@@ -139,8 +132,7 @@ const emits = defineEmits<{
   (eventName: 'selectOption', value: PossibleOption): void
 }>()
 
-const step1Modal = ref(false)
-const step2Modal = ref(false)
+const optOutModal = ref(false)
```

Comment:
Personal preference kinda thang (take it or leave it) I like to start booleans with a verb, i.e. `showOptOutModal`. It is a bit more wordy tho

Reply:
Agreed, prefer that as well. Changing.